features/bookmarks/bookmark_search.feature

```diff
@@ -230,13 +230,13 @@ Feature: Search Bookmarks
     Given "someuser" has bookmarks of works in various languages
     # reload search page to bring new language-id mappings for dropdown
     When I reload the page
-      And I select "English" from "Work language"
+      And I select "Deutsch" from "Work language"
```

Comment:
Potentially stupid question: why'd we switch which language we're testing with?

Reply:
I wanted to know if things work for languages other than the Archive's default.

Reply:
Makes sense to me! :+1: